astropy/constants/tests/test_pickle.py

```diff
@@ -10,11 +10,8 @@
                             system='si'),
              const.h,
              const.e.si]
-xfails = [False, False, False]
 
 
-@pytest.mark.parametrize(("original", "xfail"), zip(originals, xfails))
-def test_new_constant(pickle_protocol, original, xfail):
-    if xfail:
-        pytest.xfail()
+@pytest.mark.parametrize("original", originals)
+def test_new_constant(pickle_protocol, original):  # noqa
```

Comment:
Out of curiosity, why the `noqa`? I thought test files were excluded anyway...

Reply:
It is complaining about redefinition of unused "pickle_protocol". :woman_shrugging:

Reply:
It is actually weird that it is in the `test_new_constant` definition, since the code below should just take whatever was imported. Have you tried removing it? (from the function definition)

Reply:
I am not familiar with such `pytest` fixture magic. If you say to remove it, I'll try.

Reply:
Nope, looks like you need to pass it in; can't remove it -- https://travis-ci.org/astropy/astropy/jobs/598740888

I am dropping the second commit.

Reply:
OK, don't understand it really either, but probably it is some type of fixture as well. Will merge, since this cleans things up nicely anway.